Runtime/Scripts/VideoStreamTrack.cs

```diff
@@ -184,6 +194,19 @@ public override void Dispose()
             }
             base.Dispose();
         }
+
+        internal void OnVideoFrameResize(int width, int height)
```

Comment:
@doctorpangloss 
Is your concern that this method indicate that you are creating a RenderTexture?

Reply:
yes, this is going to leak textures if the width and height change.

Reply:
thanks
I understand and fixed in this commit ad70ecc